packages/sidecar/src/contextlines.ts

```diff
@@ -124,67 +124,52 @@ function isValidSentryStackFrame(frame: SentryStackFrame): frame is ValidSentryS
   return !!frame.filename && !!frame.lineno && !!frame.colno;
 }
 
-export function contextLinesHandler(req: IncomingMessage, res: ServerResponse) {
-  // We're only interested in handling a PUT request
-  if (req.method !== "PUT") {
-    res.writeHead(405);
-    res.end();
-    return;
-  }
+export const contextLinesHandler: Handler = async c => {
```

Comment:
blocker: What is `c`? Let's use more descriptive variable/parameter names. Or feel free to use desctructuring in parameters.